samples/snippets/src/main/java/pubsublite/spark/SimpleRead.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pubsublite.spark;
+
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.Row;
+import org.apache.spark.sql.SparkSession;
+import org.apache.spark.sql.streaming.OutputMode;
+import org.apache.spark.sql.streaming.StreamingQuery;
+import org.apache.spark.sql.streaming.Trigger;
+
+public class SimpleRead {
+
+  public static void main(String[] args) throws Exception {
+
+    final String appId = UUID.randomUUID().toString();
+    final String sourceSubscriptionPath = args[0];
+
+    SparkSession spark =
+        SparkSession.builder()
+            .appName(String.format("Simple read (ID: %s)", appId))
+            .master("yarn")
+            .getOrCreate();
+
+    // Read messages from Pub/Sub Lite
+    Dataset<Row> df =
+        spark
+            .readStream()
+            .format("pubsublite")
+            .option("pubsublite.subscription", sourceSubscriptionPath)
+            .load();
+
+    // Write messages to Console Output
+    StreamingQuery query =
+        df.writeStream()
+            .format("console")
+            .outputMode(OutputMode.Append())
+            .trigger(Trigger.ProcessingTime(1, TimeUnit.SECONDS))
+            .start();
+    query.awaitTermination(60 * 1000); // 60s
```

Comment:
I would suggest adding a comment here explaining why we are doing this, since the user's probably won't want to do this in the code.